tox.ini

```diff
@@ -1,9 +1,8 @@
 [tox]
-envlist = py27,py33,py34,py35,pypy,pypy3
+envlist = py27,py34,py35,py36,pypy,pypy3
 
 [testenv]
 commands =
-    python setup.py -q test -q
-deps = 
-    zope.testing
-    six
+    zope-testrunner --test-path=src
+deps =
+    .[test]
```

Comment:
Separate PR, since that environment didn't already exist?

Hmm, now that I think about it, there's only one version-specific or implementation-specific block in this whole package, and that can easily be done away with, making the trivial invocation of `coverage run -m zope.testrunner` correct and accurate across all versions. I'll add that to #8.